packages/dropdowns/src/elements/Combobox/Combobox.spec.tsx

```diff
@@ -215,9 +219,10 @@ describe('Combobox', () => {
         </Dropdown>
       );
       const combobox = getByTestId('combobox');
+      const input = combobox.querySelector('input')!;
 
-      await user.type(combobox, '{arrowup}');
-      await user.type(combobox, '{enter}');
+      await user.type(input, '{arrowup}');
+      await user.type(input, '{enter}');
```

Comment:
This test case has been questionable. I was only able to get it passing when targeting the input in the `Combobox` and the only test that has been "altered". When I tested manually, this behavior didn't behave as expected - nothing happened. Is there anything historical to note with this test case?